docs/project/engineering-note-allocation-status-performance.md

```
@@ -491,3 +491,930 @@ future.get(5, TimeUnit.SECONDS);  // 블로킹 호출
+
+#### 문제: "언제 캐시를 지울 것인가?"
+
+```java
+// 간단해 보이지만...
+@CacheEvict(value = "snapshot", key = "#matchId + ':' + #blockId")
+public void hold(Long matchId, Long blockId, Long seatId) { }
+```
+
+#### 복잡해지는 경우들
+
+**Case 1: 여러 곳에서 데이터 변경**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)